addon/-private/properties/collection/main.js

```diff
@@ -106,6 +106,22 @@ if (typeof (Symbol) !== 'undefined' && Symbol.iterator) {
   }
 }
 
+function proxyIt(instance) {
+  return new window.Proxy(instance, {
+    get: function(target, name) {
+      if (typeof(name) === 'number' || typeof(name) === 'string') {
```

Comment:
This is needed because `parseInt` blows up when called with a `Symbol` object (which we use to make it iterable).